examples/todo/src/__tests__/acceptance/todo.acceptance.ts

```diff
@@ -67,6 +67,15 @@ describe('TodoApplication', () => {
       .expect(422);
   });
 
+  it('rejects requests to create a todo with an id', async () => {
```

Comment:
Any suggestions for a better place to test this fix are appreciated 🙇‍♀️

Reply:
Maybe something like `rejects requests with input that contains excluded properties`?